pinot-core/src/main/java/com/linkedin/pinot/core/query/executor/ServerQueryExecutorV1Impl.java

```diff
@@ -93,15 +94,31 @@ public DataTable processQuery(ServerQueryRequest queryRequest, ExecutorService e
     if (schedulerWaitTimer != null) {
       schedulerWaitTimer.stopAndRecord();
     }
+    long querySchedulingTimeMs = TimeUnit.NANOSECONDS.toMillis(System.nanoTime() - timerContext.getQueryArrivalTimeNs());
```

Comment:
timercontext has time that has been obtained from getCurrentMillis (after your recent PR), right? If so, we should not be doing math between Nanotime and millis, right?